gym/spaces/box.py

```diff
@@ -56,6 +57,8 @@ def __init__(
             shape = low.shape  # type: ignore
         elif not np.isscalar(high):
             shape = high.shape  # type: ignore
+        elif np.isscalar(low) and np.isscalar(high):
```

Comment:
This makes sense, but doesn't belong in this PR

Reply:
True, I will add it to another PR